package/utils/audit/Makefile

```diff
@@ -28,14 +26,68 @@ PKG_INSTALL:=1
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/host-build.mk
 
+define Package/audit/Default
+  TITLE:=Audit Daemon
+  URL:=http://people.redhat.com/sgrubb/audit/
+  DEPENDS:=@(aarch64||arm||powerpc64||i386||x86_64)
+endef
+
+define Package/audit/Default/description
+  The audit package contains the user space utilities for
+  storing and searching the audit records generated by
+  the audit subsystem in the Linux 2.6 kernel
+endef
+
 define Package/libaudit
+$(call Package/audit/Default)
+  SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=Linux Auditing Framework (shared library)
   URL:=http://people.redhat.com/sgrubb/audit/
 endef
 
 define Package/libaudit/description
-	This package contains the audit shared library.
+$(call Package/audit/Default/description)
+  This package contains the audit shared library.
+endef
+
+define Package/libauparse
+$(call Package/audit/Default)
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE+= (parsing shared library)
+  DEPENDS+= +libaudit
+endef
+
+define Package/libauparse/description
+$(call Package/audit/Default/description)
+  This package contains the audit parsing shared library.
+endef
+
+define Package/audit-utils
+$(call Package/audit/Default)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE+= (utilities)
+  DEPENDS+= +libaudit +libauparse
+endef
+
+define Package/audit-utils/description
+$(call Package/audit/Default/description)
+  This package contains the audit utilities.
+endef
+
+define Package/audit
+$(call Package/audit/Default)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE+= (daemon)
+  DEPENDS+= +libaudit +libauparse +audit-utils +libev +@KERNEL_AUDIT
```

Comment:
This seem to depend on `libev` while we only have `libev2` in openwrt.git.

Reply:
I may mix this up and libev is unrelated to libevent2, in that case we have to migrate the package as well.

Reply:
This should be the https://github.com/openwrt/packages/blob/master/libs/libev/Makefile

I have also read in in the history that audit does have its own libev implementation if there is no libev found.
https://github.com/openwrt/packages/commit/55d588c9244f571315e942b9058c4a388dae4437#diff-1bb48de85a1017766fa1099efbdced6849929bdf1f42f256d83f8e03c78c8ed4

Reply:
@dangowrt can you comment here? I remember you weren't to keen to have libev in core?